Source/WTF/wtf/Ref.h

```diff
@@ -276,6 +276,18 @@ inline bool is(const Ref<ArgType, PtrTraits>& source)
     return is<ExpectedType>(source.get());
 }
 
+template<typename Target, typename Source, typename PtrTraits>
+inline Target& downcast(Ref<Source, PtrTraits>& source)
+{
+    return static_cast<Target&>(source.get());
+}
+
+template<typename Target, typename Source, typename PtrTraits>
+inline Target& downcast(const Ref<Source, PtrTraits>& source)
```

Comment:
Here `Target&` instead of `const Target&` is fine because Ref::get() returns a `T&` when called on a const Ref.